packages/eas-cli/src/commands/update/index.ts

```diff
@@ -562,17 +574,28 @@ export default class UpdatePublish extends EasCommand {
       Errors.error('--group and --republish flags are deprecated', { exit: 1 });
     }
 
+    const skipBundler = flags['skip-bundler'] ?? false;
+    let emitBuildMeta = flags['emit-build-meta'] ?? false;
+
+    if (skipBundler && emitBuildMeta) {
+      emitBuildMeta = false;
+      Log.warn(
+        'ignoring flag --emit-build-meta as metadata cannot be generated when skipping bundle generation'
```

Comment:
```suggestion
        'Ignoring --emit-build-meta flag as metadata cannot be generated when skipping bundle generation'
```

Reply:
I've applied this change too (and replaced the flag with new one)